delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/OrganizationsRoutesSpec.scala

```diff
@@ -271,6 +293,93 @@ class OrganizationsRoutesSpec extends BaseRouteSpec with IOFromMap {
         }
       }
     }
+
+    "undeprecate an organisation" in {
+      val org = thereIsADeprecatedOrganization
+
+      Put(s"/v1/orgs/$org/undeprecate?rev=2") ~> as(userWithWritePermission) ~> routes ~> check {
+        status shouldEqual StatusCodes.OK
+        response.asJson should not(be(deprecated))
+      }
+
+      latestRevisionOfOrganization(org) should not(be(deprecated))
+    }
+
+    "fail to undeprecate an organisation without organizations/write permission" in {
+      val org = thereIsADeprecatedOrganization
+
+      Put(s"/v1/orgs/$org/undeprecate?rev=2") ~> routes ~> check {
+        response.shouldBeForbidden
+      }
```

Comment:
Not sure it's too important but I would have a preference for using the user with org/read permission instead of anonymous

Reply:
Each of the other users only have one permission, so this should be safe